src/routes/info/privacy-policy/policy.js

```diff
@@ -3,7 +3,7 @@ export default `
 <li><a href="https://www.google.com/policies/privacy/" target="_blank" rel="noopener"><span>Google Play Services</span></a></li>
 <li><a href="https://gameanalytics.com/privacy" target="_blank" rel="noopener"><span>GameAnalytics</span></a></li>
 <li><a href="https://godotengine.org/privacy-policy" target="_blank" rel="noopener"><span>Godot</span></a></li>
-</ul><p><span>The privacy policies of each of these third parties are linked above.</span></p><h2 id="Changes-to-This-Privacy-Policy" data-id="Changes-to-This-Privacy-Policy"><span>Changes to This Privacy Policy</span></h2><p><span>We may update our Privacy Policy from time to time. The updated version will be indicated by an updated “Revised” date and the updated version will be effective as soon as it is accessible. Thus, you are advised to review this page periodically for any changes. We will notify you of any changes by posting the new Privacy Policy on this page.</span></p><h2 id="Contact-Us" data-id="Contact-Us"><span>Contact Us</span></h2><p><span>If you have any questions or suggestions about our Privacy Policy, do not hesitate to contact us via email at:</span></p><p><span>android-support@csma.technology,<br>ios-support@csma.technology,</span></p><p><span>or by post at:</span></p><p><span>CSMA Technology, LLC</span><br>
+</ul><p><span>The privacy policies of each of these third parties are linked above.</span></p><h2 id="Changes-to-This-Privacy-Policy" data-id="Changes-to-This-Privacy-Policy"><span>Changes to This Privacy Policy</span></h2><p><span>We may update our Privacy Policy from time to time. The updated version will be indicated by an updated “Revised” date and the updated version will be effective as soon as it is accessible. Thus, you are advised to review this page periodically for any changes. We will notify you of any changes by posting the new Privacy Policy on this page.</span></p><h2 id="Contact-Us" data-id="Contact-Us"><span>Contact Us</span></h2><p><span>If you have any questions or suggestions about our Privacy Policy, do not hesitate to contact us via email at:</span></p><p><span>blend-support@csma.technology,</span></p><p><span>or by post at:</span></p><p><span>CSMA Technology, LLC</span><br>
```

Comment:
not that urgent lol but i noticed the emails were old in the privacy policy that absolutely no one has ever read but i updated it lol